docs/development.md

```diff
@@ -119,6 +119,54 @@ recommended, since it requires a lot of effort to manually install all the tool
 natively on your machine and keeping them up to date to the exact same version
 specified in Docker.
 
+## rootless Docker
+
+In order to run Docker without root privileges, follow the guide at
+https://docs.docker.com/engine/security/rootless/ .
+
+Below is a quick summary of the relevant steps.
+
+1. If you have an existing version of Docker running as root, uninstall that
+   first.
+
+   ```
```

Comment:
Looks like there should be a `bash` here as welll.

Reply:
Done